Find a functionally equivalent code:android.widget.AbsListView.invokeOnItemScrollListener:COMMENT
Method Modifier: 
Comment:/**
 * Notify our scroll listener (if there is one) of a change in scroll state
 */

@@ -1,6 +1,6 @@
 {
-    if (mFastScroller != null) {
-        mFastScroller.onScroll(mFirstPosition, getChildCount(), mItemCount);
+    if (mFastScroll != null) {
+        mFastScroll.onScroll(mFirstPosition, getChildCount(), mItemCount);
     }
     if (mOnScrollListener != null) {
         mOnScrollListener.onScroll(this, mFirstPosition, getChildCount(), mItemCount);

